nextcord/application_command.py

```diff
@@ -1771,6 +1776,9 @@ def get_payload(self, guild_id: Optional[int]) -> dict:
                 #  the default (True) to ensure payload parity for comparisons.
                 ret["dm_permission"] = True
 
+        if self.nsfw:
```

Comment:
might as well use `bool` over `Optional[bool]` as `None` and `False` are removed here anyway

Reply:
But it's not really an issue.

Reply:
i didnt say it was, it was just a suggestion

Reply:
I see

Reply:
Stalled per [this message](https://canary.discord.com/channels/881118111967883295/990633056207523860/991811066822676592) in [this thread](https://canary.discord.com/channels/881118111967883295/990633056207523860) on the [Nextcord server](https://discord.com/invite/nextcord)

Reply:
Since Discord publicly announced the release date of this feature, I think it's best to unstall this review comment.

Reply:
I think that at least `self.nsfw` should explicitly check `None` just to be sure.
```suggestion
        if self.nsfw is not None:
```